homeassistant/components/notify/html5.py

```diff
@@ -131,29 +140,79 @@ def post(self, request):
         return self.json_message("Push notification subscriber registered.")
 
 
+class HTML5PushCallbackView(HomeAssistantView):
```

Comment:
Your comment on the pivotal story
> Actions are also already in the iOS app, could we possibly build a unified system in the notifications component for actions?

So now I'm confused, this seems like an HTML5  specific implementation instead of a generic notify implementation ?